cmd/rm-main.go

```diff
@@ -264,6 +293,43 @@ func removeRecursive(url string, isIncomplete bool, isFake bool, older int) erro
 	return nil
 }
 
+func isHostName(url string) bool {
+	parts := strings.Split(url, "/")
+	return len(parts) == 1
+}
+
+// removes all the buckets recursively.
+func removeSiteRecursive(url string, isIncomplete bool, isFake bool, older int) (err error) {
+	clnt, pErr := newClient(url)
+	if pErr != nil {
+		errorIf(pErr.Trace(url), "Failed to remove `"+url+"` recursively.")
```

Comment:
This is not appropriate error message here.

Reply:
why?

Reply:
This is fine because of `pErr.Trace(url)`.  Sorry for the confusion.